Content munge devportal (#11982)
hashicorp/packer: website/content/docs/provisioners/index.mdx

# Provisioners

Provisioners use built-in and third-party software to install and configure the
machine image after booting. Provisioners prepare the system, so you may want to use them for the following use cases:

- installing packages
- patching the kernel
- creating users
- downloading application code

Refer to the [`provisioner`](/docs/templates/hcl_templates/blocks/build/provisioner) block documentation to learn more

> about working with provisioners. The documentation includes a page for each type of provisioner.

Author: ```suggestion
about working with provisioners. The documentation includes details about each type of provisioner.
```
